src/types.ts

```diff
@@ -251,6 +251,17 @@ export interface MOptions {
   styles?: true | 'none' | 'sass' | {
     configFile: string
   }
+  /**
+   * Disable the modern SASS compiler and API.
+   *
+   * The module will check for `sass` and `sass-embedded` dev dependencies:
+   * - if `sass` and `sass-embedded` dev dependencies are installed, the module will configure the modern SASS compiler.
+   * - if only `sass` dev dependency is installed, the module will configure the modern SASS API and will enable []().
+   * - otherwise will configure the legacy SASS compiler.
+   *
+   * @default false
+   */
+  disableModernSassCompiler?: boolean
```

Comment:
cleanup and update it: Vite 6 will use `modern-compiler` as the default https://github.com/vitejs/vite/blob/2c3c3b252b0f1c403652c3f558080348f26b32f2/packages/vite/src/node/plugins/css.ts#L2392-L2394